monitoring/api/UptimeCheckSample/Program.cs

```diff
@@ -0,0 +1,212 @@
+            string displayName)
+        {
+            // Define a new config.
+            var config = new UptimeCheckConfig()
+            {
+                DisplayName = displayName,
+                MonitoredResource = new MonitoredResource()
+                {
+                    Type = "uptime_url",
+                    Labels = { { "host", hostName } }
+                },
+                HttpCheck = new UptimeCheckConfig.Types.HttpCheck()
+                {
+                    Path = "/",
+                    Port = 80,
+                },
+                Timeout = TimeSpan.FromSeconds(10).ToDuration(),
+                Period = TimeSpan.FromMinutes(5).ToDuration()
+            };
+            // Create a client.
+            var client = UptimeCheckServiceClient.Create();
+            string projectName = new ProjectName(projectId).ToString();
+            // Create the config.
+            var newConfig = client.CreateUptimeCheckConfig(projectName, config);
+            Console.WriteLine(newConfig.Name);
+            return 0;
+        }
+        // [END monitoring_uptime_check_create]
+
+        // [START monitoring_uptime_check_delete]
+        public static object DeleteUptimeCheckConfig(string configName)
+        {
+            var client = UptimeCheckServiceClient.Create();
+            client.DeleteUptimeCheckConfig(configName);
+            Console.WriteLine($"Deleted {configName}");
+            return 0;
+        }
+        // [END monitoring_uptime_check_delete]
+
+
+        // [START monitoring_uptime_check_list_configs]
+        public static object ListUptimeCheckConfigs(string projectId)
+        {
+            var client = UptimeCheckServiceClient.Create();
+            var configs = client.ListUptimeCheckConfigs(
+                new ProjectName(projectId).ToString());
+            foreach (UptimeCheckConfig config in configs)
+            {
+                Console.WriteLine(config.Name);
+            }
+            return 0;
+        }
+        // [END monitoring_uptime_check_list_configs]
+
+        // [START monitoring_uptime_check_list_ips]
+        public static object ListUptimeCheckIps()
+        {
+            var client = UptimeCheckServiceClient.Create();
+            var ips = client.ListUptimeCheckIps(new ListUptimeCheckIpsRequest());
+            foreach (UptimeCheckIp ip in ips)
+            {
+                Console.WriteLine("{0,20} {1}", ip.IpAddress, ip.Location);
+            }
+            return 0;
+        }
+        // [END monitoring_uptime_check_list_ips]
+
+        // [START monitoring_uptime_check_update]
+        public static object UpdateUptimeCheck(string configName,
+            string newHttpPath, string newDisplayName)
+        {
+            var client = UptimeCheckServiceClient.Create();
+            var config = client.GetUptimeCheckConfig(configName);
+            var fieldMask = new FieldMask();
+            if (newDisplayName != null)

```

Comment:
Should we remove the input parameters validation or add it in other snippets to make it more consistent across the sample code?